library/spdm_crypt_lib/libspdm_crypt_cert.c

```diff
@@ -1147,11 +1149,13 @@ bool libspdm_x509_common_certificate_check(const uint8_t *cert, size_t cert_size
         goto cleanup;
     }
 
-    /* 10. verify spdm defined extension*/
-    status = libspdm_verify_leaf_cert_spdm_extension(cert, cert_size,
-                                                     is_requester_cert, cert_model);
-    if (!status) {
-        goto cleanup;
+    if (!(set_cert && cert_model == SPDM_CERTIFICATE_INFO_CERT_MODEL_ALIAS_CERT)) {
```

Comment:
I don't think we want to verify this for SPDM_CERTIFICATE_INFO_CERT_MODEL_GENERIC_CERT, right?

Maybe change to `cert_model != SPDM_CERTIFICATE_INFO_CERT_MODEL_DEVICE_CERT`

Reply:
Correct! I forgot about generic certs. Fixed!